Sources/Testcontainers/Docker/Models/ImageParams.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ImageParams.swift
+//
+//
+//  Created by cristian on 3/09/24.
+//
+
+import Foundation
+
+public struct ImageParams {
+    
+    public let name: String
+    public let tag: String
+    public let src: String?
```

Comment:
what is src? 🤔

Reply:
I read from the documentation, and just put it as a new optional param, but I did not have the chance to give a try how it works. I think it is better to remove it for now, to avoid mistakes.